pkg/blob/blob.go

```diff
@@ -415,7 +415,7 @@ func (d *Driver) GetAuthEnv(ctx context.Context, volumeID, protocol string, attr
 		case storageAuthTypeField:
 			azureStorageAuthType = v
 			authEnv = append(authEnv, "AZURE_STORAGE_AUTH_TYPE="+v)
-		case storageIentityClientIDField:
+		case storageIdentityClientIDField:
```

Comment:
`storageIdentityClientID` is only for blobfuse mount on the agent node since user may grant managed identity permission to the agent node, cannot be used in volume cloning since volume cloning happens in control plane side.

Reply:
btw, could you send a separate PR for this typo issue? thanks.

Reply:
send a separate pr for this issue: https://github.com/kubernetes-sigs/blob-csi-driver/pull/1157